tests/client.reconnect.test.js

```diff
@@ -95,7 +95,7 @@ describe('When radar server restarts', function() {
   it('reestablishes presence', function(done) {
     this.timeout(4000);
     var verifySubscriptions = function() {
-      client2.presence('restore').get(function(message) {
+      client.presence('restore').get(function(message) {
```

Comment:
Before I made changes, this test used to randomly break.  We can talk about it; I believe the test setup had a problem in its configuration.